diff --git a/BlackHole/BlackHole.c b/BlackHole/BlackHole.c
index ac516ef..897fd8c 100644
--- a/BlackHole/BlackHole.c
+++ b/BlackHole/BlackHole.c
@@ -2543,7 +2543,7 @@ static OSStatus	BlackHole_GetDevicePropertyData(AudioServerPlugInDriverRef inDri
 			//	any 32 bit integer, but common values for this property are defined in
 			//	<CoreAudio/AudioHardwareBase.h>
 			FailWithAction(inDataSize < sizeof(UInt32), theAnswer = kAudioHardwareBadPropertySizeError, Done, "BlackHole_GetDevicePropertyData: not enough space for the return value of kAudioDevicePropertyTransportType for the device");
-			*((UInt32*)outData) = kAudioDeviceTransportTypeVirtual;
+			*((UInt32*)outData) = kAudioDeviceTransportTypeUSB;
 			*outDataSize = sizeof(UInt32);
 			break;
 
diff --git a/BlackHole/BlackHole.plist b/BlackHole/BlackHole.plist
index bcbc8d6..8862fe8 100644
--- a/BlackHole/BlackHole.plist
+++ b/BlackHole/BlackHole.plist
@@ -24,14 +24,14 @@
 	<string>596</string>
 	<key>CFPlugInFactories</key>
 	<dict>
-		<key>e395c745-4eea-4d94-bb92-46224221047c</key>
+		<key>4a91361d-19af-4ba0-a87a-0a6242ed6ae4</key>
 		<string>BlackHole_Create</string>
 	</dict>
 	<key>CFPlugInTypes</key>
 	<dict>
 		<key>443ABAB8-E7B3-491A-B985-BEB9187030DB</key>
 		<array>
-			<string>e395c745-4eea-4d94-bb92-46224221047c</string>
+			<string>4a91361d-19af-4ba0-a87a-0a6242ed6ae4</string>
 		</array>
 	</dict>
 </dict>
